runtime/storage/storage-provider-base.js

```diff
@@ -13,7 +13,7 @@ import util from '../recipe/util.js';
 
 export default class StorageProviderBase {
   constructor(type, arc, name, id, key) {
-    var trace = tracing.start({cat: 'view', name: 'InMemoryStorageProvider::constructor', args: {type: type.key, name: name}});
+    var trace = tracing.start({cat: 'view', name: 'StorageProviderBase::constructor', args: {type: type.key, name: name}});
```

Comment:
Are we using `var` here and below intentionally? I see it in a bunch of other runtime code where `const` or `let` seem would be preferred.

Reply:
> Are we using var here and below intentionally? I see it in a bunch of other runtime code where const or let seem would be preferred.

It's not intentional, we just haven't cared to review the use of var/let/const to date. Now that eslint is set up, we could turn on the `no-var` check, but would need to update the existing code or figure out how to only lint the changes.